### PR TITLE
FireseedNetwork.Theseus2000 version 1.2

### PR DESCRIPTION
**Pull request by seedy60**
- [ ] Have you signed the [Contributor License Agreement](https://cla.opensource.microsoft.com/microsoft/winget-pkgs)?
- [ ] Have you checked that there aren't other open [pull requests](https://github.com/microsoft/winget-pkgs/pulls) for the same manifest update/change?
- [ ] Have you [validated](https://github.com/microsoft/winget-pkgs/blob/master/AUTHORING_MANIFESTS.md#validation) your manifest locally with `winget validate --manifest <path>`? 
- [ ] Have you tested your manifest locally with `winget install --manifest <path>`?
- [ ] Does your manifest conform to the [1.0 schema](https://github.com/microsoft/winget-cli/blob/master/doc/ManifestSpecv1.0.md)?

Note: `<path>` is the name of the directory containing the manifest you're submitting.

-----

###### Microsoft Reviewers: [Open in CodeFlow](https://portal.fabricbot.ms/api/codeflow?pullrequest=https://github.com/microsoft/winget-pkgs/pull/46943)

### CODE REVIEW

File: manifests/f/FireseedNetwork/Theseus2000/1.2/FireseedNetwork.Theseus2000.locale.en-US.yaml [hunk 1]
```diff
@@ -0,0 +1,13 @@
+# Created using wingetcreate 0.5.0.1
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.1.0.schema.json
+
+PackageIdentifier: FireseedNetwork.Theseus2000
+PackageVersion: 1.2
+PackageLocale: en-US
+Publisher: Fireseed Network
+PackageName: Theseus2000
+License: Freeware
+ShortDescription: Theseus2000 Setup
```
Comment (OfficialEsco) [inline manifests/f/FireseedNetwork/Theseus2000/1.2/FireseedNetwork.Theseus2000.locale.en-US.yaml:10]: Guess you missed this one from the old PR
```suggestion
ShortDescription: The high powered networking suite for PCs!
```

File: manifests/f/FireseedNetwork/Theseus2000/1.2/FireseedNetwork.Theseus2000.installer.yaml [hunk 2]
```diff
@@ -0,0 +1,13 @@
+# Created using wingetcreate 0.5.0.1
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.1.0.schema.json
+
+PackageIdentifier: FireseedNetwork.Theseus2000
+PackageVersion: 1.2
+Installers:
+- Architecture: x86
```
Comment (OfficialEsco) [inline manifests/f/FireseedNetwork/Theseus2000/1.2/FireseedNetwork.Theseus2000.installer.yaml:7]: And again its probably 64 bit
```suggestion
- Architecture: x64
```

**Review by OfficialEsco — CHANGES_REQUESTED** [review]

**Review by OfficialEsco — APPROVED** [review]

**Comment by lechacon** [comment]
Looks like the validation failed because the app needs to be run from its root directory.

@SeedyTeevee , you are the developer for this app, right? Can you confirm if it would make sense for it to be failing like that in the pipeline?

The output was:
```
Checking update server status...
Checking for updates
Firing up
Traceback (most recent call last):
  File "theseus2000.py", line 22, in <module>
  File "t2000.pyx", line 1209, in t2000.CheckNet
  File "t2000.pyx", line 1194, in t2000.CheckUpd
  File "t2000.pyx", line 1093, in t2000.CheckStart
  File "t2000.pyx", line 57, in t2000.start
  File "site-packages\playsound.py", line 35, in _playsoundWin
  File "site-packages\playsound.py", line 31, in winCommand
playsound.PlaysoundException:
    Error 275 for command:
        open "snd/startup.mp3" alias playsound_0.51567388909
    Cannot find the specified file.  Make sure the path and filename are correct.
[3348] Failed to execute script theseus2000
```
So it's looking for `snd/startup.mp3`, which won't work unless running from `C:\Seedy\Theseus2000\`.

**Comment by SeedyTeevee** [comment]
I don't understand why this is happening, it's never happened to me before. How is the program being installed? The installer should just extract the program files to whereever you choose, and you should be able to run the program from there.

**Comment by lechacon** [comment]
From what I remember when I looked into this, the executable would fail unless it was run from its install directory. The shortcut created would set the working directory to the right place, so I would be okay with completing the PR, but the [website hosting the installer](https://www.houseoffireseed.ml) seems to be down.

> How is the program being installed?

The installer is downloaded from the provided URL and executed normally.

> you should be able to run the program from there.

The validation does not know where to run the program from. It's running it from some other directory than where it installs to, which causes it to fail.

@denelon I don't know if we already do this, but for the validation of executables, I think we could try also running the shortcuts found in case a program requires specific arguments or working directory given in a shortcut.

**Comment by SeedyTeevee** [comment]
@lechacon When you say the website is down, what do you mean. There's nothing down about the site from where I am; I can download the installer just fine. HTTP 200 all round.

**Comment by lechacon** [comment]
Weird. On my computer I'm getting error DNS_PROBE_FINISHED_NXDOMAIN, but on my phone I can reach the site without problems. Looks related to DNS?

**Comment by SeedyTeevee** [comment]
I can access www.houseoffireseed.ml and houseoffireseed.ml from my PC just fine; no DNS issues at all. Try clearing your browser cache/cookies.

**Review by ghost — APPROVED** [review]

**Pull request merged**